libraries/AC_Autorotation/AC_Autorotation.cpp

```diff
@@ -251,54 +321,121 @@ float AC_Autorotation::get_rpm(bool update_counter)
     return current_rpm;
 }
 
+void AC_Autorotation::initial_flare_estimate()
+{
+    //estimate hover thrust
+    float _col_hover_rad = radians(_col_min + (_col_max - _col_min)*_col_hover);
+    float b = _param_solidity*6.28f;
+    _disc_area=3.14*sq(_param_diameter*0.5f);
+    float lambda = (-(b/8.0f) + safe_sqrt((sq(b))/64.0f +((b/3.0f)*_col_hover_rad)))*0.5f;
+    float freq=_governed_rpm/60.0f;
+    float tip_speed= freq*6.28f*_param_diameter*0.5f;
```

Comment:
2 pi ? We have a define for that too.
```suggestion
    float tip_speed= freq * M_2PI *_param_diameter * 0.5;
```

You could also just use pi drop the 0.5 at the end, but the compiler should be able to figure that out.

Reply:
done